app/src/main/res/values/colors.xml

```diff
@@ -91,8 +91,8 @@
     <color name="contacts_dark">#C8AAEE</color>
 
     <!-- TODO: Use correct colors for AI -->
```

Comment:
Is this `TODO` still relevant ?

Reply:
Oh nice catch, I thought I had removed it